CMakeLists.txt

```diff
@@ -86,6 +87,8 @@ if(MSVC)
   add_definitions(-DNNVM_EXPORTS)
   add_definitions(-DDMLC_STRICT_CXX11)
   add_definitions(-DNOMINMAX)
+  set(SUPPORT_F16C FALSE)
```

Comment:
This question might be stupid, but why are we disabling support on Windows? If this is a processor instruction, why does it depend on the OS?

Reply:
It's actually the MSVC (visual studio compiler) that's not yet supported. That doesn't support the current usage of f16c functions. Future work. Updated the message. 
To use the feature we need not just processor, but also compiler support.

Reply:
Aaah I see, that makes sense. Thanks for elaborating